backend/models.js

```diff
@@ -1,5 +1,23 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
+const bcrypt = require('bcrypt');
+
+const UserSchema = new Schema({
+  email: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  password: {
+    type: String,
+    required: true,
+  }
+});
+
+// TODO: Hash passwords
+UserSchema.methods.verifyPassword = function (password) {
+  return password == this.password;
```

Comment:
Should this be a triple equals? AFAIK === is usually preferred in JS. == automatically converts type, so just want to check